### DISCUSSION TITLE
Windows desktop app

### DISCUSSION THREAD

**Discussion by kayanco** [ente-io/ente#286] [Enhancements]
Hi,

Any plans to add a Windows desktop app (like Authy Desktop version)?

**Comment by vishnukvmd** [comment]
Hey, this is being tracked here: https://github.com/ente-io/auth/issues/243

**Comment by mnvr** [comment]
Closing since this is a duplicate of https://github.com/ente-io/ente/discussions/510 (also, the desktop app is now in beta, close to a release, so this is sort of done-ish anyway!)
